katas/args/alfredo/ModuleOne.js

```diff
@@ -0,0 +1,74 @@
+import { Schema } from './Schema.js';
+export class ModuleOne {
```

Comment:
Hi @AlfredPereira , maybe the comment was misunderstood. ModuleArgs was a good name and that one should have been kept, and rename the file `module1.test.js`.

Anyway, I will merge now since we need to close this kata. but keep in mind good practices about naming for your next code